openpgp/integration_tests/utils_test.go

```diff
@@ -145,7 +146,7 @@ func randFileHints() *openpgp.FileHints {
 	return &openpgp.FileHints{
 		IsBinary: mathrand.Intn(2) == 0,
 		FileName: string(fileName),
-		ModTime: time.Now(),
+		ModTime:  time.Now(),
```

Comment:
Could you please revert the unrelated whitespace changes in this PR, to make the diff slightly cleaner? Or if you want you can make a separate PR with those first, and then rebase this on top once that's merged.